### ISSUE TITLE
Evaluating the model

### ISSUE THREAD

**Issue by Nikh1l** [cfotache/pytorch_custom_yolo_training#4]
Can you please provide us code to evaluate the model? Or some guidance regarding the same?

Thank you

**Comment by bsharma2** [comment]
Is loss.item() giving us the training loss or the validation loss?

**Comment by aedirn** [comment]
> Is loss.item() giving us the training loss or the validation loss?

@bsharma2 Training loss. It looks like this repo doesn't do validation at all, I think?
